org.kde.lokalize.json

```diff
@@ -0,0 +1,70 @@
+{
+    "id": "org.kde.lokalize",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "lokalize",
+    "rename-icon": "lokalize",
+    "finish-args": 
+    [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kross",
+            "buildsystem": "cmake-ninja",
+            "config-opts": 
+            [ 
+                "-DENABLE_TESTING=OFF"
+            ],
+            "sources": 
+            [ 
+                { 
+                    "type": "git",
+                    "url": "https://invent.kde.org/frameworks/kross.git",
+                    "branch": "v5.69.0"
+                }
+            ]
+        },
+        {
+            "name": "hunspell",
+            "sources": 
+            [
+                { 
+                    "type": "git",
+                    "url": "https://github.com/hunspell/hunspell.git"
```

Comment:
what version here? you need to pin to a specific version, not just use master.

Reply:
https://github.com/flathub/flathub/pull/1603/commits/5cd008c9be83162308f1a0b7d32c91eb853ae475